tc_shortener/handlers/shortener.py

```diff
@@ -82,6 +82,7 @@ def post(self, **kwargs):
                 shortener.put(key, url)
 
                 self.write(json.dumps({'key': key}))
+                self.set_header("Content-Type", "application/json")
```

Comment:
Given http://tornado.readthedocs.org/en/latest/web.html#tornado.web.RequestHandler.write maybe we can just do `self.write(dict(key=key))` ?